envs/example/allinone-rhel7/group_vars/all.yml

```diff
@@ -0,0 +1,111 @@
+---
+stack_env: example_allinone
+floating_ip: 172.16.0.100
+xtradb:
+  sst_auth_password: asdf
+
+percona:
+  replication: False
+
+common:
+  hwraid:
+    enabled: False
+  system_tools:
+    mcelog: True
+    lm_sensors_package: lm_sensors
+    packages:
+      - acl
+      - curl
+      - dstat
+      - ethtool
+      - git
+      - iotop
+      - ltrace
+      - mtr
+      - nc
+      - ruby-devel
+      - ruby
+      - socat
+      - tree
+      - vim
+      - sysstat
+      - logrotate
+      - ntp
+      - lvm2
+      - cracklib
+      - mlocate
+  python:
+    system_packages:
+      - python-pip
```

Comment:
Which yum repo is `python-pip` package in? I can find it even I enable all redhat repo mentioned in this PR.